src/components/base/BaseButton.vue

```diff
@@ -8,8 +8,10 @@ const props = withDefaults(
     | "primary-rev"
     | "secondary"
     | "tertiary"
-    | "tertiary-rev";
-    buttonSize: "small" | "regular" | "medium" | "large";
+    | "tertiary-rev"
+    | "breadcrumb"
+    | "breadcrumb-rev";
```

Comment:
consider passing rev as a separate prop as it seems it is separate parameter that can apply to all button types

Reply:
I will try this on - not occurred to me